tasks/setup-Debian.yml

```diff
@@ -13,5 +13,21 @@
     state: present
     update_cache: yes
 
+- name: Download specific Jenkins version.
+  get_url:
+    url: "{{ jenkins_pkg_url }}/jenkins_{{ jenkins_version }}_all.deb"
+    dest: "/tmp/jenkins.deb"
+  when: jenkins_version is defined
+
+- stat: path="/tmp/jenkins.deb"
+  register: specific_version
+
+- name: Install correct version of Jenkins is installed.
```

Comment:
The name is a little confusing... maybe `Install correct version of Jenkins if .deb file exists.`?